### DISCUSSION TITLE
Same page errs with hrefs, copying the $locale 2 times

### DISCUSSION THREAD

**Discussion by bitdom8** [ivanhofer/typesafe-i18n#327] [Q&A] [answered]
In the latest sveltekit, there might be an error. Is it typesafe related or svelte?

    "@sveltejs/adapter`
-auto": "next",
    "@sveltejs/kit": "1.0.0-next.345",
    "@types/cookie": "^0.5.1",
    "@typescript-eslint/eslint-plugin": "5.26.0",
    "@typescript-eslint/parser": "5.26.0",
    "eslint": "8.16.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-svelte3": "^4.0.0",
    "prettier": "2.6.2",
    "prettier-plugin-svelte": "2.7.0",
    "svelte": "3.48.0",
    "svelte-check": "^2.7.1",
    "svelte-preprocess": "4.10.6",
    "tslib": "^2.4.0",
    "typescript": "4.7.2"
```

When I'm on the same page
`<a href="{$locale}/tags" class="nav-link">`

The link becomes:
`<a href="{$locale}/{$locale}/tags" class="nav-link">`

When I click so it refers to http://localhost:3000/en/tags/tags

**Comment by ivanhofer** [comment, marked as answer]
Sounds like a `SvelteKit` bug. You could try to replace `$locale` with a `writable` store and see if the error still occurs.

**Reply by bitdom8** [reply]
That didn't work. Will try to write full url for each however it's not the best solution

**Reply by ivanhofer** [reply]
Ah, now I see it. Yes, you should always stick to absolute urls. It makes the links easier to predict and maintain. Unless you have a specific use-case and know what you do.

**Comment by bitdom8** [comment]
@ivanhofer you are the one of the best contributor on coding. Thanks Спасибо. It works now

**Reply by ivanhofer** [reply]
Thanks for your kind words! ❤️
